src/main/java/model/UMLClassManager.java

```diff
@@ -413,34 +466,47 @@ public int removeRelationship(String srcClass, String type, String destClass) {
 		return ErrorHandler.setCode(0);
 	}
 	
+	/**
+	 * Lists relationships of the entire model
+	 */
+	public ArrayList<ArrayList<String[]>> printRelationships() {
+		if(relationships.isEmpty()) {
+			return null;
+		}
+		else {
+			ArrayList<ArrayList<String[]>> result = new ArrayList<ArrayList<String[]>>();
+			for(Map.Entry<String, UMLRelationship> entry : relationships.entrySet()) {
+				String name2 = entry.getValue().getClass2().getName();
+				for(Map.Entry<String, UMLRelationship> r : relationships.entrySet()) {
+					if(!name2.equals(r.getValue().getClass1().getName())) {
+						result.add(printRelationships(r.getValue().getClass1().getName()));
+					}
+				}
+			}
+			return result;
+		}
+	}
+	
 	/**
 	 * List the relationships the given class has
 	 * @param className
 	 * @return 'tuple' of [string of relationships, return code];
 	 */
-	public Object[] listRelationships(String className) {

```

Comment:
Not for future: Rewrite this method to return just an array of the relationships. Similar to the fields and methods.